packages/webapp/src/components/modals/collectionModal.tsx

```diff
@@ -0,0 +1,24 @@
+import { useRouter } from 'next/router';
+
+export const CollectionGameModal = () => {
+
+  const router = useRouter();
+  const handleClick = () => {
+    router.push('/collection');
+  };
```

Comment:
You can inline this: `<label ... onClick={() => router.push('/collection')}`.

In fact it might be better to use a Next.js `Link` instead of `label`. More on this below.